src/SFA.DAS.Admin.Roatp.Web/Models/Session/AddProviderSessionModel.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace SFA.DAS.Admin.Roatp.Web.Models.Session;
+public class AddProviderSessionModel
+{
+    public int Ukprn { get; set; }
+    public string? LegalName { get; set; } = string.Empty;
```

Comment:
LegalName should not be null and should not be initialised, just mark the property as required.
`public required string LegalName {get; set;}`